backend-rust/src/migrations/m0016-passive-delegation.sql

```diff
@@ -0,0 +1,23 @@
+CREATE INDEX passive_delegators_idx
+ON accounts (delegated_target_baker_id, delegated_stake)
+WHERE delegated_target_baker_id IS NULL AND delegated_stake != 0;
+
+CREATE TABLE passive_delegation_payday_commission_rates(
```

Comment:
I would need this table as well to calculate the reward metrics

Reply:
I actually need the history. Could you just store the history?

Reply:
After discussion it does not seem as if these are included into the rewards table. For now we are doing the same.